areas/appservice/src/AzureMcp.AppService/Commands/Database/DatabaseAddCommand.cs

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using AzureMcp.AppService.Models;
+using AzureMcp.AppService.Options;
+using AzureMcp.AppService.Options.Database;
+using AzureMcp.AppService.Services;
+using AzureMcp.Core.Commands;
+using AzureMcp.Core.Models.Option;
+using AzureMcp.Core.Services.Telemetry;
+using Microsoft.Extensions.Logging;
+
+namespace AzureMcp.AppService.Commands.Database;
+
+public sealed class DatabaseAddCommand(ILogger<DatabaseAddCommand> logger) : BaseAppServiceCommand<DatabaseAddOptions>()
+{
+    private const string CommandTitle = "Add Database to App Service";
+    private readonly ILogger<DatabaseAddCommand> _logger = logger;
+
+    public override string Name => "add";
+
+    public override string Description =>
+        """
+        Add a database connection to an App Service. This command configures database connection
+        settings for the specified App Service, allowing it to connect to a database server.
+        """;
```

Comment:
Have you tried running the utility in `eng\tools\ToolDescriptionConfidenceScore` to see if the description is good enough? You can find instructions on how to do that [here](https://github.com/Azure/azure-mcp/blob/main/eng/tools/ToolDescriptionConfidenceScore/README.md) :)